src/zrc/api/serializers.py

```diff
@@ -278,13 +280,8 @@ class Meta:
                 'validators': [NotSelfValidator(), HoofdzaakValidator()],
             },
             'relevante_andere_zaken': {
-                'child': serializers.URLField(
-                    label=_("URL naar andere zaak"),
-                    max_length=255,
-                    validators=[URLValidator(
-                        get_auth=get_auth,
-                        headers={'Content-Crs': 'EPSG:4326', 'Accept-Crs': 'EPSG:4326'}
-                    )]
+                'child': serializers.DictField(
```

Comment:
Please create a dedicated `AndereZaakSerializer` with the `url` field and the `aard_relatie` field. This serializer should then be specified as `many=True` instead of messing with `DictField`